nanowallet/Jenkinsfile

```diff
@@ -0,0 +1,6 @@
+defaultCiPipeline {
+	platform = ['ubuntu']
+	ciBuildDockerfile = 'node10.Dockerfile'
```

Comment:
why node10 instead of javascript?

Reply:
NanoWallet requires node 10 to build.  Javascript image is currently at node 16.

Reply:
Its using ``gulp-cli`` for tasks which seems to only with the older version of nodes.
There are a lot of depreciated modules due to node 10 requirements.

Reply:
I also asked about updating the node version in discord - https://discord.com/channels/856325968096133191/865180563433979904/1100437664227921940